### PR TITLE
Add google optomise and job submit event

### LINKED ISSUE #336: [career improvements] Add ab testing framework to site

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Add google optomise script to all pages
- Add job submit event on /careers/##### pages

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/careers/2114847
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that there is an event on the submit button and google optomise is running on the page

## Issue / Card

Fixes #336

### CODE REVIEW

File: templates/careers/job-detail.html [hunk 1]
```diff
@@ -89,7 +89,12 @@ <h3>Apply for this role</h3>
           <button type="button" class="js-locate-me">Locate me</button>
           <span class="js-locate-error u-hide" style="color: #c7162b">Location unavailable</span>
           <hr style="margin: 1rem 0;" />
-          <button type="submit" class="p-button--positive u-no-margin--bottom">Submit Application</button>
+          <button type="submit" class="p-button--positive u-no-margin--bottom" onsubmit="dataLayer.push({
```
Comment (anthonydillon) [inline templates/careers/job-detail.html:92]: The `onsubmit` event is triggered from the form element, not the button element.

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Comment in the code

File: templates/careers/job-detail.html (hunk 1 above)
Comment (pmahnke) [reply templates/careers/job-detail.html]: Moved to an `onclick`

**Review by anthonydillon — APPROVED** [review]
LGTM :+1:

**Pull request merged**